test/sanity/run-test.sh

```diff
@@ -50,4 +59,4 @@ _output/smbplugin --endpoint "$endpoint" --nodeid "$nodeid" -v=5 &
 
 echo 'Begin to run sanity test...'
 readonly CSI_SANITY_BIN='csi-sanity'
```

Comment:
This works for `prow-jobs` (pull-csi-driver-smb-sanity) but not for `GitHub workflow` (Linux Unit tests / Build (pull_request) ).